catalogue/__init__.py

```diff
@@ -22,10 +23,14 @@ def create(*namespace: str, entry_points: bool = False) -> "Registry":
     """
     if check_exists(*namespace):
         raise RegistryError(f"Namespace already exists: {namespace}")
-    return Registry(namespace, entry_points=entry_points)
+    
+    if generic_type is None:
```

Comment:
This isn't exactly right...but I couldn't figure out how to get mypy to accept a direct default for the argument